src/graphql/operations/leaderboards.ts

```diff
@@ -0,0 +1,58 @@
+import db from '../../helpers/mysql';
+import log from '../../helpers/log';
+import { buildWhereQuery, checkLimits, formatLeaderboard } from '../helpers';
+import { capture } from '@snapshot-labs/snapshot-sentry';
+
+export default async function (parent, args) {
+  const { first, skip, where = {} } = args;
+
+  checkLimits(args, 'leaderboards');
+
+  const fields = {
+    user: 'string',
+    space: 'string',
+    vote_count: 'number',
+    proposal_count: 'number'
+  };
+  const whereQuery = buildWhereQuery(fields, 'l', where);
+  const defaultOrder = 'l.vote_count DESC, l.proposal_count DESC';
+
+  const orderBy = Object.keys(fields).includes(args.orderBy)
+    ? args.orderBy
+    : null;
+  let orderDirection = (args.orderDirection || 'desc').toUpperCase();
+  if (!['ASC', 'DESC'].includes(orderDirection)) orderDirection = 'DESC';
+
+  const query = `
+    SELECT l.*,
+      spaces.settings,
+      spaces.flagged as spaceFlagged,
+      spaces.verified as spaceVerified,
+      spaces.turbo as spaceTurbo,
+      spaces.hibernated as spaceHibernated,
```

Comment:
Why you need this here?

Reply:
Those are properties of the `Space` object.

First version of this PR was returning result like 

```
"space": "ens.eth",
"user": "0xcEF0b1BbEC22eDC3D07Eac2F5f1053359A96A5e2",
"vote_count": 38,
"proposal_count": 0
```

Now, it's returning 

```
{
        "user": {
          "id": "0xcEF0b1BbEC22eDC3D07Eac2F5f1053359A96A5e2",
          "ipfs": "bafkreiai7c3canwhltsxjtc4ba2fli4mlrn7plmo4nl27cp27cugzsxl44",
          "created": 1671060995,
          "name": "",
          "about": "",
          "avatar": "ipfs://bafkreiguy3xfr246zcucsnk7lvneck74jb5rp3nvcqapfbveutwwimspo4"
        },
        "space": {
          "id": "ens.eth",
          "activeProposals": 0,
          "verified": true
        },
        "vote_count": 38,
        "proposal_count": 0
      },
```

with `space` and `user` returning object instead of just the ID, to match the API on sx

Reply:
I think `space` in leaderboard could be just string, else we would make it slow like other queries. in most of the times we would query leaderboard of single space right? 

same with `user` but could be helpful, is it possible to compare times with and without these two tables here?

Reply:
`space` and `user` now returned as simple string, avoiding all `JOIN` in the sql query